src/main/java/uk/gov/mint/CTHandler.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.mint;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.store.EntriesUpdateDAO;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+
+public class CTHandler implements Handler {
+    private final CanonicalJsonMapper canonicalJsonMapper;
+    private final String register;
+    private final String ctserver;
+    private final Client client;
+    private final EntryValidator entryValidator;
+
+    public CTHandler(String register, String ctserver, Client client, EntryValidator entryValidator) {
```

Comment:
Is it a good idea to create client inside the constructor so that if an object is created than we are sure that the client is not null and we don't check again in processEntries?